plugin.py

```diff
@@ -572,6 +580,10 @@ def onDeviceRemoved(self, Unit):
                 devicetype = 'light'
             elif Device.Type == 0xf1:       # pTypeColorSwitch
                 devicetype = 'light'
+            elif (Device.Type == 0xf4 and   # pTypeGeneralSwitch
+                Device.SubType == 0x49 and  # sSwitchGeneralSwitch
+                ((Device.SwitchType == 3) or (Device.SwitchType == 15) or (Device.SwitchType == 13))):   # Blind
```

Comment:
Please break in three lines and add comment for each line which STYPE 3, 15 and 13 corresponds to.